scm/github.go

```diff
@@ -670,7 +659,7 @@ func (s *GithubSCM) RemoveMember(ctx context.Context, opt *OrgMembershipOptions)
 func (s *GithubSCM) GetUserScopes(ctx context.Context) *Authorization {
 	// Authorizations.List method will always return nil, response struct and error,
```

Comment:
Comment not update; seems to describe only method.

Reply:
Fixed.